crates/proof-of-sql/src/base/database/column_operation_error.rs

```diff
@@ -53,6 +54,13 @@ pub enum ColumnOperationError {
         /// The underlying source error
         source: DecimalError,
     },
+
+    #[snafu(display("Unsupported binary operator: {}", operator))]
```

Comment:
Same issue as in Unary

Reply:
Waiting for UnaryOp to get merged and rebase upon it to get done this